src/Cli/FrameworkCommands.php

```diff
@@ -299,7 +299,7 @@ protected function nextWpVersionThatExists($current_version)
 
         // If our versions do not match, check that the latest version exists before continuing.
         // Greater than comparision to ensure that we don't get older versions. See https://github.com/pantheon-systems/cos-framework-clis/pull/76#issuecomment-1118650316
-        if ($latest_version > $current_version) {
+        if (version_compare($latest_version, $current_version)) {
```

Comment:
```suggestion
        if (version_compare($latest_version, $current_version, ">")) {
```
I think in a conditional we either want the third arg, or `version_compare()>= 0`. I prefer the third arg.